docs/UserGuide.adoc

```diff
@@ -42,7 +103,7 @@ You may use create a folder called `Codeducator` on your `Desktop`, or in `My Do
 
 .  Double-click the file to start the app. You should see the application open in a appear in a few seconds.
 +
-image::Ui.png[width="790"]
+image::firstStart.png[width="790"]
```

Comment:
I think we have to change this to UI.png since the graders are going to look at this file.